Code/GraphMol/FindStereo.cpp

```diff
@@ -367,72 +380,111 @@ std::string getBondSymbol(const Bond *bond) {
     auto aidx = atom->getIdx();
+    atomSymbols[aidx] = getAtomCompareSymbol(*atom);
     if (detail::isAtomPotentialStereoAtom(atom)) {
       auto sinfo = detail::getStereoInfo(atom);
       switch (sinfo.specified) {
         case Chirality::StereoSpecified::Unknown:
+          knownAtoms.set(aidx);
+          atomSymbols[aidx] += std::to_string(aidx);
+          break;
         case Chirality::StereoSpecified::Specified:
           knownAtoms.set(aidx);
+          if (sinfo.descriptor == StereoDescriptor::Tet_CCW) {
+            atomSymbols[aidx] += "_CCW";
+          } else if (sinfo.descriptor == StereoDescriptor::Tet_CW) {
+            atomSymbols[aidx] += "_CW";
+          } else {
+            atomSymbols[aidx] += "_@";
+          }
           break;
         case Chirality::StereoSpecified::Unspecified:
+          if (flagPossible) {
+            possibleAtoms.set(aidx);
+            if (!cleanIt) {
+              atomSymbols[aidx] += "_" + std::to_string(aidx);
+            }
+          }
           break;
         default:
           throw ValueErrorException("bad StereoInfo.specified type");
       }
-      if (flagPossible ||
-          sinfo.specified != Chirality::StereoSpecified::Unspecified) {
-        possibleAtoms.set(aidx);
-        // set "fake stereo"
-        ochiralTypes[aidx] = atom->getChiralTag();
-        atom->setChiralTag(Atom::CHI_TETRAHEDRAL_CW);
-        atomSymbols[aidx] = (boost::format("%d%s_%d") % atom->getIsotope() %
-                             atom->getSymbol() % aidx)
-                                .str();
-      } else {
-        atomSymbols[aidx] = getAtomCompareSymbol(*atom);
-      }
-    } else {
-      atomSymbols[aidx] = getAtomCompareSymbol(*atom);
-      if (cleanIt) {
-        atom->setChiralTag(Atom::ChiralType::CHI_UNSPECIFIED);
-      }
+    } else if (cleanIt) {
+      atom->setChiralTag(Atom::ChiralType::CHI_UNSPECIFIED);
     }
   }
+}
 
+void initBondInfo(ROMol &mol, bool flagPossible, bool cleanIt,
+                  boost::dynamic_bitset<> &knownBonds,
+                  std::vector<std::string> &bondSymbols,
+                  boost::dynamic_bitset<> &possibleBonds) {
+  for (const auto bond : mol.bonds()) {
+    auto bidx = bond->getIdx();
+    bondSymbols[bidx] = getBondSymbol(bond);
+    if (detail::isBondPotentialStereoBond(bond)) {
+      auto sinfo = detail::getStereoInfo(bond);
+      switch (sinfo.specified) {
+        case Chirality::StereoSpecified::Unknown:
+          knownBonds.set(bidx);
+          bondSymbols[bidx] += "_" + std::to_string(bidx);
+          break;
+        case Chirality::StereoSpecified::Specified:
+          knownBonds.set(bidx);
+          if (sinfo.descriptor == StereoDescriptor::Bond_Cis) {
+            bondSymbols[bidx] += "_cis";
+          } else if (sinfo.descriptor == StereoDescriptor::Bond_Trans) {
+            bondSymbols[bidx] += "_trans";
+          } else {
+            bondSymbols[bidx] += "_chi";
```

Comment:
This is the same as with chiralities, isn't it? Just a mark to indicate this is neither "cis" nor "trans", kind of a "not yet supported" stereo type. Would it make sense to use the same mark as for "non CW nor CCW" in chiralities above?